src/promptbase/utils/consts.py

```diff
@@ -0,0 +1,29 @@
+BIGBENCH_SUBJECTS = [
```

Comment:
Not a huge point, but if this is only for bigbench use, shouldn't this be in the bigbench directory?

Reply:
I originally put it here because I reference it in the __main__.py file, but I think it makes sense to move it to a consts file in bigbench